components/camel-platform-http-vertx/src/main/java/org/apache/camel/component/platform/http/vertx/VertxPlatformHttpSupport.java

```diff
@@ -119,6 +127,15 @@ static Object toHttpResponse(HttpServerResponse response, Message message, Heade
         return body;
     }
 
+    static Integer determineContentLength(Exchange camelExchange, Object body) {
+        if (body instanceof byte[]) {
```

Comment:
What if the body is just a String shouldnt we do a length check on that too?

Reply:
I didn't want to mess with encoding tbh..